prisma/schema.prisma

```diff
@@ -155,3 +157,24 @@ model PasswordToken {
     userId String @unique
 
 }
+
+model Centre {
+    id Int @id @default(autoincrement())
+
+    name            String @unique
+    displayText     String
+    displaySequence Int
+    streetAddress   String
+    city            String
+    stateProvince   String
+    country         String
+
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    uniqueKey String?  @unique
+    updatedBy String?
+    createdBy String?
+
+    // Relation fields
+    users Member[]
```

Comment:
I think changing the users to member would be less confusing while querying later on, since we have user table also

Reply:
Yes, agree